yt_dlp/extractor/afreecatv.py

```diff
@@ -19,7 +19,7 @@
 
 
 class AfreecaTVBaseIE(InfoExtractor):
-    _NETRC_MACHINE = 'afreecatv'
+    _NETRC_MACHINE = 'soop'
```

Comment:
not sure about changing this

Reply:
```suggestion
    _NETRC_MACHINE = 'afreecatv'
```
we can change in the future, for now let's keep it the same to not break users' netrc files